app/Actions/Photo/Pipes/Standalone/PlacePhoto.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+namespace App\Actions\Photo\Pipes\Standalone;
+
+use App\Actions\Diagnostics\Pipes\Checks\BasicPermissionCheck;
+use App\Contracts\Image\StreamStats;
+use App\Contracts\PhotoCreate\StandalonePipe;
+use App\DTO\PhotoCreate\StandaloneDTO;
+use App\Enum\SizeVariantType;
+use App\Exceptions\ConfigurationException;
+use App\Exceptions\Handler;
+use App\Exceptions\MediaFileOperationException;
+use App\Image\StreamStat;
+use App\Models\Configs;
+
+class PlacePhoto implements StandalonePipe
+{
+	public function handle(StandaloneDTO $state, \Closure $next): StandaloneDTO
+	{
+		// Create target file and symlink/copy/move source file to target.
+		// If import strategy request to delete the source file.
```

Comment:
```suggestion
		// If the import strategy requests to delete the source file
```

Mostly the "." as it looks like this is meant to run into the next line - i.e. `$this->sourceFile` is only deleted if the import strategy requests it.